dotcom-rendering/src/web/components/RichLink.tsx

```diff
@@ -21,7 +21,7 @@ import { Avatar } from '@frontend/web/components/Avatar';
 interface Props {
 	richLinkIndex: number;
 	cardStyle: RichLinkCardType;
-	imageData: RichLinkImageData;
+	imageData?: RichLinkImageData;
```

Comment:
By making this optional we allow for the variation that is wanted in blogs where the image is not shown